python/train.py

```diff
@@ -18,61 +18,70 @@
 from __future__ import print_function
 
 import math
+from ffcc import model
+from ffcc import ops
+from ffcc import io
+from ffcc import input as ffcc_input
+import os
+import json
+from tensorboard.plugins.hparams import api as hp
+import tensorflow as tf
 
-from . import model
-from . import ops
-import input as ffcc_input
-import tensorflow.compat.v1 as tf
 
-# This directory stores the TFRecord for training.
-DATA_DIR = ''
+os.environ['CUDA_VISIBLE_DEVICES'] = '-1'
+
+
+# This directory stores the images and metadata for training.
+DATA_DIR = '../data/shi_gehler/preprocessed/GehlerShi'
+
+# Project name
+PROJECT_NAME = 'GehlerShi'
 
 # Model dir for checkpoints, etc.
-MODEL_DIR = ''
+MODEL_DIR = './GehlerShi_model/'
```

Comment:
Can this be just 'GehlerShi_model', since it's relative to the current directory by default?